### PR TITLE
Notify user about recorded user interaction

### PR DESCRIPTION
**Pull request by aryangupta701**
- added notification feature

### CODE REVIEW

**Comment by aryangupta701** [comment]
ready for review

File: source/ContentScript/recorder.ts [hunk 1]
```diff
@@ -351,6 +354,68 @@ class Recorder {
...
+      await this.waitForNotificationToClear();
+    }
+    const floatingDiv = document.getElementById('ZapfloatingDiv');
+    if (!floatingDiv) {
+      console.log('Floating Div Not Found !');
+      return;
+    }
+
+    this.isNotificationRaised = true;
+    const messageElement = document.createElement('p');
```
Comment (thc202) [inline source/ContentScript/recorder.ts:389]: Should keep using the same `className` to avoid recording events when the message is showing?

Comment (aryangupta701) [reply source/ContentScript/recorder.ts:389]: yess

Comment (aryangupta701) [reply source/ContentScript/recorder.ts:389]: fixed

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**